lib/rules/selector-combinator-space-after/__tests__/index.js

```diff
@@ -315,87 +373,131 @@ testRule(rule, {
   reject: [
     {
       code: "a+ a {}",
+      fixed: "a+a {}",
       description: "space after + combinator",
       message: messages.rejectedAfter("+"),
       line: 1,
       column: 2
     },
     {
       code: "a> a {}",
+      fixed: "a>a {}",
       description: "space after > combinator",
       message: messages.rejectedAfter(">"),
       line: 1,
       column: 2
     },
     {
       code: "a~ a {}",
+      fixed: "a~a {}",
       description: "space after ~ combinator",
       message: messages.rejectedAfter("~"),
       line: 1,
       column: 2
     },
     {
       code: "a+\na{}",
+      fixed: "a+a{}",
       description: "newline after + combinator",
       message: messages.rejectedAfter("+"),
       line: 1,
       column: 2
     },
     {
       code: "a+\r\na{}",
+      fixed: "a+a{}",
       description: "CRLF after + combinator",
       message: messages.rejectedAfter("+"),
       line: 1,
       column: 2
     },
     {
       code: "a>\na{}",
+      fixed: "a>a{}",
       description: "newline after > combinator",
       message: messages.rejectedAfter(">"),
       line: 1,
       column: 2
     },
     {
       code: "a>\r\na{}",
+      fixed: "a>a{}",
       description: "newline after > combinator",
       message: messages.rejectedAfter(">"),
       line: 1,
       column: 2
     },
     {
       code: "a~\na{}",
+      fixed: "a~a{}",
       description: "newline after ~ combinator",
       message: messages.rejectedAfter("~"),
       line: 1,
       column: 2
     },
     {
       code: "a~\r\na{}",
+      fixed: "a~a{}",
       description: "CRLF after ~ combinator",
       message: messages.rejectedAfter("~"),
       line: 1,
       column: 2
     },
     {
       code: "a + .foo.bar ~a {}",
+      fixed: "a +.foo.bar ~a {}",
       description: "multiple combinators: space after + combinator",
       message: messages.rejectedAfter("+"),
       line: 1,
       column: 3
     },
     {
       code: "#foo +.foo.bar ~ a {}",
+      fixed: "#foo +.foo.bar ~a {}",
       description: "multiple combinators: no space after ~ combinator",
       message: messages.rejectedAfter("~"),
       line: 1,
       column: 16
     },
     {
       code: "a >>> a {}",
+      fixed: "a >>>a {}",
       description: "space after >>> combinator",
       message: messages.rejectedAfter(">>>"),
       line: 1,
       column: 3
+    },
+    {
+      code: "a/*comment*/ > /*comment*/a {}",
+      fixed: "a/*comment*/ >/*comment*/a {}",
+      description: "comment",
+      message: messages.rejectedAfter(">"),
+      line: 1,
+      column: 14
+    },
+    {
+      code: "a/*comment*/ > /*comment*/a, b/*comment*/ > /*comment*/b {}",
+      fixed: "a/*comment*/ >/*comment*/a, b/*comment*/ >/*comment*/b {}",
+      description: "comment",
+      message: messages.rejectedAfter(">"),
+      line: 1,
+      column: 14
+    },
+    {
```

Comment:
Can you move this and the following tests into the `syntax: scss` `testRule` block at the bottom of the file, please?